lib/watcher.js

```diff
@@ -61,6 +61,8 @@ function Watcher(logger, api, files, sources) {
 		this.busy = api.run(specificFiles || files, {
 			runOnlyExclusive: runOnlyExclusive
 		}).then(function (runStatus) {
+			runStatus.testRunElapsed = runStatus.testRunStart && process.hrtime(runStatus.testRunStart);
```

Comment:
Is this suggestion due to `process` not being supported in the browser or is there another reason? I'm just asking, because I'm curious.

My rationale for choosing `hrtime` over `Date` was providing the most amount of information to the reporters. One can always get milliseconds from nanoseconds, but the reverse is not true.